Containerfile

```diff
@@ -4,44 +4,53 @@ MAINTAINER LSIT Systems <lsitops@lsit.ucsb.edu>
 
 USER root
 
-RUN apt update -qq && apt install -yq nano wget && apt-get clean
+ENV TZ America/Los_Angeles
+RUN apt update && \
+    apt install -yq nano wget && \
+    apt clean && \
+    ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
```

Comment:
Let's put this as a separate RUN since it's a separate general operation from package install.